app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -36,50 +39,68 @@ import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.viewinterop.AndroidView
 import ch.hikemate.app.R
 import ch.hikemate.app.model.route.HikeRoute
 import ch.hikemate.app.model.route.ListOfHikeRoutesViewModel
-import ch.hikemate.app.ui.components.AppropriatenessMessage
 import ch.hikemate.app.ui.components.BackButton
+import ch.hikemate.app.ui.components.ElevationGraph
+import ch.hikemate.app.ui.map.HikeDetailScreen.MAP_MAX_ZOOM
+import ch.hikemate.app.ui.map.HikeDetailScreen.MAP_MIN_ZOOM
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_ADD_DATE_BUTTON
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_BOOKMARK_ICON
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_DETAIL_ROW_TAG
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_DETAIL_ROW_VALUE
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_ELEVATION_GRAPH
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_HIKE_NAME
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_MAP
+import ch.hikemate.app.ui.map.HikeDetailScreen.TEST_TAG_PLANNED_DATE_TEXT_BOX
 import ch.hikemate.app.ui.navigation.NavigationActions
 import ch.hikemate.app.ui.navigation.Screen
-import ch.hikemate.app.utils.calculateBestZoomLevel
-import ch.hikemate.app.utils.getGeographicalCenter
+import ch.hikemate.app.utils.MapUtils.calculateBestZoomLevel
+import ch.hikemate.app.utils.MapUtils.getGeographicalCenter
+import ch.hikemate.app.utils.MapUtils.showHikeOnMap
```

Comment:
Change: instead of importing those functions directly, import `MapUtils` and use `MapUtils.function()`. It improves code clarity.

Reply:
done